build_extra/rust/rust.gni

```diff
@@ -170,7 +170,7 @@ template("rust_crate") {
       "rustc",
       rebase_path(source_root, root_build_dir),
       "--env-out-dir",
-      rebase_path(root_out_dir),
+      rebase_path(out_dir, root_out_dir),
```

Comment:
Should probably rebased with "//src" as the second arg.

Reply:
Or with `//` and add an extra `..` to the `include!`

Reply:
Fixed.